locale/ru/LC_MESSAGES/reference/reference_lua/box_index/pairs.po

```diff
@@ -34,14 +34,15 @@ msgstr ""
 "все кортежи в порядке по возрастанию или по убыванию, начиная с указанного "
 "ключа. Однако другие типы индексов не поддерживают упорядочивание."
 
+#, fuzzy
```

Comment:
This update breaks translations in a lot of places. For example, page https://docs.d.tarantool.io/ru/doc/restructure-docs-1/reference/reference_lua/box_index/pairs/

Reply:
Looks like make update-po worked in an unexpected way. I can just remove this commit. In this case, what is the correct way to batch update translations?